src/script/auth/page/SingleSignOnForm.tsx

```diff
@@ -108,18 +108,16 @@ const SingleSignOnForm = ({
       return;
     }
     codeInput.current.value = codeInput.current.value.trim();
+    const currentValidationError = codeInput.current.checkValidity()
```

Comment:
How about destructuring `codeInput.current`? Might be worth it since it is used quite a lot here.